shopperz/api/tests.py

```diff
@@ -1,3 +1,69 @@
 from django.test import TestCase
+from .models import User
+from rest_framework import status
+from .serializers import UserSerializer
+from rest_framework.test import APITestCase, APIClient
+from django.urls import reverse
 
 # Create your tests here.
+
+
+class CreateUser(APITestCase):
+    '''
+       This is use to test for register user route
+       
+    '''
+    
+    
+    def setUp(self):
+        self.user = {
+            'firstname': 'john',
+            'lastname': 'tunde',
+            'email': 'john@gmail.com',
+            'password': 'john',
+            'phonenumber': '08085963214'
+        }
+        
+        # self.same_email = {
+        #     'firstname': 'fola',
+        #     'lastname': 'shade',
+        #     'email': 'john@gmail.com',
+        #     'password': 'john',
+        #     'phonenumber': '08085963214'
+        # }
+        
+        self.missing_email = {
+            'firstname': 'john',
+            'lastname': 'tunde',
+            'email': '',
+            'password': 'john',
+            'phonenumber': '08085963214'
+        }
+        
+        self.empty_details = {
+            'firstname': '',
+            'lastname': '',
+            'email': '',
+            'password': '',
+            'phonenumber': ''
+        }
+    
+    def test_create_user(self):
```

Comment:
We can make this test better by also testing for the response object for each of the test cases.